### PR TITLE
Polish Cassandra tutorial

### PR DESCRIPTION
**Pull request by cody-clark**
* Template applied
* Per [issue #7](kubernetes/examples#7) and [issue #10](kubernetes/examples#10) Replication Controller and DaemonSet sections removed.
* Regarding line 118: the original tutorial linked [here](https://kubernetes.io/docs/user-guide/kubectl/v1.7/#edit) and the only other page I could find was from [v1.4](https://kubernetes-v1-4.github.io/docs/user-guide/kubectl/kubectl_edit/). Should I leave out the further reading?

Draft, placeholder links, etc do not merge

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4422)

### CODE REVIEW

**Comment by ahmetb** [comment]
@cody-clark Can you please submit the tutorial to https://github.com/kubernetes/examples/ instead? That's where we host the tutorial content now, and we have a script to manually import it to this repository.

Sorry about the misleading state. We now have a DO NOT EDIT notice in the cassandra.md file to prevent further edits as of late last week.

We can still have the this reviewed by sig-docs folks for this in k/examples repository, just make sure you @{mention} people you need a review from.

File: docs/tutorials/stateful-application/cassandra.md [hunk 1]
```diff
@@ -1,374 +1,134 @@
 ---
-title: "Example: Deploying Cassandra with Stateful Sets"
+title: Deploying Cassandra a StatefulSet
```
Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: "Deploying Cassandra a StatefulSet" is really difficult to parse. "Using StatefulSets with Cassandra" perhaps?

File: docs/tutorials/stateful-application/cassandra.md [hunk 2]
```diff
@@ -1,374 +1,134 @@
...
-
-If you want to jump straight to the commands we will run,
-here are the steps:
-
-```sh
-#
-# StatefulSet
-#
+{% capture overview %}
+This tutorial will show you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this instance, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
```
Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Avoid future tense in docs.  Change "will show" to "shows"

File: docs/tutorials/stateful-application/cassandra.md [hunk 3]
```diff
@@ -1,374 +1,134 @@
...
+* Use a StatefulSet to create a Cassandra ring.
+* Validate the StatefulSet.
+* Modify the StatefulSet.
+* Delete the StatefulSet and its pods.
+{% endcapture %}
 
-# create a service to track all cassandra statefulset nodes
-kubectl create -f cassandra/cassandra-service.yaml
+{% capture prerequisites %}
+You should already have a basic familiarity with [Pods](#), [Services](#), and [StatefulSets](#), but to complete this tutorial you only need:
```
Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Dead links :(

Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Also, I suggest rewording this sentence.  "To complete this tutorial, you should already have a basic familiarity with Pods, Services, and StatefulSets.  In addition, you should:"
This avoid unnecessary "buts" and "onlys"

File: docs/tutorials/stateful-application/cassandra.md [hunk 4]
```diff
@@ -1,374 +1,134 @@
...
+{% endcapture %}
 
-# create a service to track all cassandra statefulset nodes
-kubectl create -f cassandra/cassandra-service.yaml
+{% capture prerequisites %}
+You should already have a basic familiarity with [Pods](#), [Services](#), and [StatefulSets](#), but to complete this tutorial you only need: 
 
-# create a statefulset
-kubectl create -f cassandra/cassandra-statefulset.yaml
+* To [Install and Configure](/docs/tasks/tools/install-kubectl/) the `kubectl` command line
```
Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Avoid leading with a preposition for better readability.  Remove the leading "to"s.

File: docs/tutorials/stateful-application/cassandra.md [hunk 5]
```diff
@@ -1,374 +1,134 @@
...
 
-# validate the Cassandra cluster. Substitute the name of one of your pods.
-kubectl exec -ti cassandra-0 -- nodetool status
+* To have a v1.5 or later Kubernetes Cluster running
```
Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Avoid stating a version number, which will make keeping this doc up to date more difficult. I suggest just stating "Have a supported Kubernetes Cluster running".

File: docs/tutorials/stateful-application/cassandra.md [hunk 6]
```diff
@@ -1,374 +1,134 @@
...
-#
-# Resource Controller Example
-#
-
-# create a replication controller to replicate cassandra nodes
-kubectl create -f cassandra/cassandra-controller.yaml
-
-# validate the Cassandra cluster. Substitute the name of one of your pods.
-kubectl exec -ti cassandra-xxxxx -- nodetool status
+Warning: [Minikube](/docs/getting-started-guides/minikube/) defaults to 1024MB of memory and 1 CPU which results in an insufficient resource errors.
```
Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Not your fault, but we need better callouts for warnings like these.  Filed the following issue: https://github.com/kubernetes/kubernetes.github.io/issues/4458

At the very least, please bold "Warning"

Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Maybe change this section to "Additional setup instructions for Minikube"?

File: docs/tutorials/stateful-application/cassandra.md [hunk 7]
```diff
@@ -1,374 +1,134 @@
...
-
-kubectl create -f cassandra/cassandra-daemonset.yaml --validate=false
-
-# resource cleanup
-kubectl delete service -l app=cassandra
-kubectl delete daemonset cassandra
 ```
+minikube start --memory 5120 --cpus=4
+``` 
+* To download [cassandra-service.yaml](docs/tutorials/stateful-application/cassandra-service.yaml) and [cassandra-statefulset.yaml](/docs/tutorials/stateful-application/cassandra-statefulset.yaml)
```
Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Dead links and a confusing sentence.  Not sure what this is asking me to do. 
Also, in general, avoid single bullet points.

File: docs/tutorials/stateful-application/cassandra.md [hunk 8]
```diff
@@ -1,374 +1,134 @@
...
 
-## Step 1: Create a Cassandra Headless Service
-
-A Kubernetes _[Service](/docs/user-guide/services)_ describes a set of
-[_Pods_](/docs/user-guide/pods) that perform the same task. In
-Kubernetes, the atomic unit of an application is a Pod: one or more containers
-that _must_ be scheduled onto the same host.
+{% capture lessoncontent %}
+## Creating a Cassandra Headless Service
+A Kubernetes [Service](/docs/user-guide/services) describes a set of [Pods](/docs/user-guide/pods) that perform the same task.
```
Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Services link should be: https://kubernetes.io/docs/concepts/services-networking/service/
Pods link should be: https://kubernetes.io/docs/concepts/workloads/pods/pod/

Things under "user-guides" are dead (for the most part) :)

File: docs/tutorials/stateful-application/cassandra.md [hunk 9]
```diff
@@ -1,374 +1,134 @@
...
+## Creating a Cassandra Headless Service
+A Kubernetes [Service](/docs/user-guide/services) describes a set of [Pods](/docs/user-guide/pods) that perform the same task. 
 
-The Service is used for DNS lookups between Cassandra Pods, and Cassandra clients
-within the Kubernetes Cluster.
+The following `Service` is used for DNS lookups between Cassandra pods and clients within the Kubernetes Cluster.
 
-Here is the service description:
+1. `cd` to the folder you saved the .yaml files.
+2. Create a Service to track all Cassandra StatefulSet Nodes from the following .yaml file:
```
Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Service -> `Service`

Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: also .yaml should be `.yaml`

File: docs/tutorials/stateful-application/cassandra.md [hunk 10]
```diff
@@ -1,374 +1,134 @@
...
-The response should be like:
+The response should be
 
 ```console
 NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
 cassandra   None         <none>        9042/TCP   45s
 ```
 
-If an error is returned the service create failed.
+If anything else returns, the service was not successfully created. Check out [Debug Services](/docs/tasks/debug-application-cluster/debug-service/) for common issues.
```
Comment (jaredbhatti) [inline docs/tutorials/stateful-application/cassandra.md]: Nitpick.  "Check out" is difficult to parse for someone who doesn't speak english as a first language.  It's especially difficult since "Check out" can mean "check out code from git".  I suggest using the word "Read" instead of the phrase "check out".  This phrase is used in other parts of the doc too.

**Comment by cody-clark** [comment]
Submitting PR to https://github.com/kubernetes/examples/
Comments applied.

**Comment by ahmetb** [comment]
Thanks @cody-clark!

**Comment by chenopis** [comment]
FYI, this PR has been superseded by https://github.com/kubernetes/examples/pull/38
